laos_gggi/combine_data.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import sys
+
+sys.path.append("..")
```

Comment:
You shouldn't need this, just use absolute import paths (e.g. `from laos_gggi.emdat_processing import process_emdat`)

Reply:
I tried this, but for some reason, it did not work.

Reply:
Solved!